running/ambassador-edge/aes.yaml

```diff
@@ -0,0 +1,332 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: ambassador
+  labels:
+    product: aes
+rules:
+  # Edge Stack mostly needs the same permissions as Ambassador OSS,
+  # but:
+  #  1. It needs extra permissions ("update"/"patch"/"create"/"delete")
+  #     for getambassador.io resources.
+  #  2. It needs extra permissions ("create"/"update") for secrets.
+  #  3. It needs permission to "create" events.
+  #  4. It needs permission to "get"/"create"/"update" Leases (or
+  #     Endpoints, if on Kubernetes <1.12).
+  - apiGroups: [""]
+    resources: ["namespaces", "services", "pods"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["getambassador.io"]
+    resources: ["*"]
+    verbs: ["get", "list", "watch", "update", "patch", "create", "delete"]
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["networking.internal.knative.dev"]
+    resources: ["clusteringresses", "ingresses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["networking.internal.knative.dev"]
+    resources: ["ingresses/status", "clusteringresses/status"]
+    verbs: ["update"]
+  - apiGroups: ["extensions", "networking.k8s.io"]
+    resources: ["ingresses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["extensions", "networking.k8s.io"]
+    resources: ["ingresses/status"]
+    verbs: ["update"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "list", "watch", "create", "update"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["get", "list", "watch", "create"]
+  - apiGroups: ["coordination.k8s.io"]
+    resources: ["leases"]
+    verbs: ["get", "create", "update"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["get", "list", "watch", "create", "update"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ambassador
+  namespace: ambassador-edge
+  labels:
+    product: aes
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: ambassador
+  labels:
+    product: aes
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: ambassador
+subjects:
+  - kind: ServiceAccount
+    name: ambassador
+    namespace: ambassador-edge
+
+######################################################################
+# Redis
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: ambassador-redis
+  namespace: ambassador-edge
+  labels:
+    product: aes
+spec:
+  type: ClusterIP
+  ports:
+    - port: 6379
+      targetPort: 6379
+  selector:
+    service: ambassador-redis
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ambassador-redis
+  namespace: ambassador-edge
+  labels:
+    product: aes
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      service: ambassador-redis
+  template:
+    metadata:
+      labels:
+        service: ambassador-redis
+    spec:
+      containers:
+        - name: redis
```

Comment:
you're running redis but haven't given it any storage. Does it need to persist across restarts or is it just using it as a cache?

Reply:
Looks like it's fine - found this in the manual:

> The Ambassador Edge Stack relies on Redis to store short-lived authentication credentials and rate limiting information. If the Redis data store is lost, users will need to log back in and all existing rate-limits would be reset.

Maybe add that to the README so the next person doesn't have the same question.

Reply:
cool added a redis section to the readme